### PR TITLE
Hedgehog Quest added (needs art implemented) + minor fixes

### PR DESCRIPTION
**Pull request by DanielTheBard**
Just need someone with grunt spritesmith (@Lefnire @Sabrecat) to do the art and it's good to go! (I think)

### CODE REVIEW

**Comment by DanielTheBard** [comment]
Pet art: https://trello-attachments.s3.amazonaws.com/52e9d25cd467121630c6350d/52e9d5da7488724b20bc95f1/08f2bae80f0e618403eee050189ca8cf/hedgehog_pets_by_TheInventrix.png

Mount art: (Uh it's separate by body and head... for every color) https://trello.com/c/CE6bN0Ip/4-quest-mounts 

Egg art: https://trello-attachments.s3.amazonaws.com/52e9d25cd467121630c6350d/52e9d5e1fe94654d2437015e/f0992dc76aa4e62fcb91af54ff76b69b/hedgehog_egg_by_TheInventrix.png

Boss art: https://trello-attachments.s3.amazonaws.com/52e9d25cd467121630c6350d/52e9d5f2efbc62a673bd454c/578f30732c12ab75d47e47fcab52f7c8/hedgeboss_by_Inventrix.png

**Comment by lefnire** [comment]
awesome man! Will hit this soon as we're releasing hedgehogs, should be this month

**Comment by DanielTheBard** [comment]
@lefnire - Awesome! Will it be out before Friday, do you think?

**Comment by lefnire** [comment]
Will try me darndest to make it happen! I'll keep you posted

**Comment by DanielTheBard** [comment]
The food correction looks major but my fork wasn't up to date. So I just deleted what was in my version of the file and pasted what's in the current version before making the change.

File: locales/en/pets.json [hunk 1]
```diff
@@ -1,35 +1,34 @@
 {
-"_commentpets" : "PETS RELATED STRINGS",
```
Comment (lefnire) [inline locales/en/pets.json:2]: I don't think we want to remove these `_comment*` lines (they're removed here, and a few times below)

Comment (lefnire) [inline locales/en/pets.json:2]: Due to spacing, you can see what I mean here https://github.com/HabitRPG/habitrpg-shared/pull/156/files?w=1

Comment (paglias) [inline locales/en/pets.json:2]: @lefnire the _comment strings have been removed from the new locales

Comment (lefnire) [inline locales/en/pets.json:2]: Ah, gotcha

**Comment by lefnire** [comment]
@DanielTheBard I cherry-picked the hedgehog-specific stuff, and I've pushed them merged with spritesmith & everything. Unfortunately, the two non-hedgehog commits I was having trouble with. They were causing merge conflicts and lots of extra code which didn't seem intended:
- https://github.com/DanielTheBard/habitrpg-shared/commit/f1585bef75dc3f7b61ac87e15b461fd05819b2ca
- https://github.com/DanielTheBard/habitrpg-shared/commit/0165e6f4c07438f1135a16cdc1182c0faae7412a

Any chance of creating a new pull request for those? I'll contrib.level you for HH

**Pull request closed without merging**